### DISCUSSION TITLE
[Script request] Pterodactyl

### DISCUSSION THREAD

**Discussion by Feramance** [community-scripts/ProxmoxVE#51] [Request script]
### Application Name

Pterodactyl

### Website

https://pterodactyl.io/

### Description

This would need two scripts in my opinion, one for the panel and another for `wings` (the node). Used to host game servers of nearly any kind

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [ ] This is not a game-related request.

**Comment by MickLesk** [comment]
Not yet or in the next time. Currently we dont create game Scripts.

**Comment by Feramance** [comment]
Given that Crafty Controller has been released, I feel this should be re-approached. Pterodactyl or Pelican, either or. And another LXC for wings

**Comment by Data-PhiIe** [comment]
Would also like ptrodactyl/pelican (would be cool if it was just one script that asks which one you want). Also would really like it to be a VM, not a LXC. LXC containers can not live migrate like VM's so a VM would be ideal to prevent service interruptions.

**Comment by bvdberg01** [comment]
Pterodactyl Panel is added with PR https://github.com/community-scripts/ProxmoxVE/pull/2801 also added Pterodactyl Wings LXC in PR https://github.com/community-scripts/ProxmoxVE/pull/2800
